src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -35,23 +34,31 @@ public static async Task<FunctionMetadataResponse> ToFunctionMetadataResponse(th
             var response = new FunctionMetadataResponse
             {
                 Name = functionMetadata.Name,
-
-                // Q: can functionMetadata.ScriptFile be null or empty?
```

Comment:
The answer to this Q was yes! For proxies, the ScriptFile can be null. I discovered this when adding tests. Is this blowing up for people in production with proxies?

I made this conditional below